CMakeLists.txt

```diff
@@ -132,8 +132,16 @@ if (WITH_LUA_ENGINE STREQUAL Lua)
 endif ()
 
 include(LuaAddExecutable)
+
+# TODO: define MINIZ_NO_STDIO for small size
```

Comment:
Is this important to do before merging?

Reply:
No, I hope to make PR to miniz to support MINIZ_NO_STDIO build with cmake later, miniz cmake without MINIZ_NO_STDIO options now. So let's merge, and fix this TODO after miniz with this feature.

Reply:
The miniz PR is https://github.com/richgel999/miniz/pull/316